### ISSUE TITLE
Generating simpleSchema for Meteor apps

### ISSUE THREAD

**Issue by JackAdams** [bshamblen/json-schema-generator#5]
Hi Brian,
This isn't an issue (so feel free to close immediately), but just thought I'd share this: https://github.com/JackAdams/constellation-schema
since it's kind of in the same space (i.e. generating schema from JS objects). In this case, I'm using docs from a Meteor collection to generate simpleSchema, using a bit of guesswork.
This package is more of a toy thing, but might amuse you for a few minutes.  Here are a couple of videos of me demoing it:
https://www.youtube.com/watch?v=ZDdtfiyXfTA
https://www.youtube.com/watch?v=5w-MnjjIgks
Cheers
Brent

**Comment by bshamblen** [comment]
Brent,

Thanks for sharing. I created a separate package to convert JSON Schema to SimpleSchema a while back, around the same time you created your project:

https://github.com/bshamblen/meteor-json-simple-schema

By using the output of this package (JSON to JSON Schema) with the other package (JSON Schema to SimpleSchema) I was able to achieve the same result. However, for our particular use case I needed to keep the logic separate, so we could simply generate JSON schemas from JSON. 

Regards,
Brian
